src/App/ResultsCount.js

```diff
@@ -1,12 +1,49 @@
+import { Select } from '@rebass/forms/styled-components'
+import { useEffect, useState } from 'react'
+
+import { Flex, Box, Text } from '../Primitives'
+import { useQueryParam } from '../router-utils'
 import { useSearchStore } from './stores'
 
+const COUNT_OPTIONS = [10, 25, 50, 100]
```

Comment:
Since our default is `50`, I would maybe change the list to:

```suggestion
const COUNT_OPTIONS = [25, 50, 100, 250]
```